### PR TITLE
use depd@2.0.0 to get rid of eval()

### PR DESCRIPTION
**Pull request by arlac77**

### CODE REVIEW

**Comment by arlac77** [comment]
breaks on node 0.6 + 0.8 can those be deprecated ?

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "license": "MIT",
   "repository": "jshttp/http-errors",
   "dependencies": {
-    "depd": "~1.1.2",
+    "depd": "^2.0.0",
```
Comment (dougwilson) [inline package.json]: Modules outside of the organization ideally should be pinned per our policy, at most the patch floated, but may as well just pin this one.

Comment (arlac77) [reply package.json]: bumped major & new history entry

Comment (dougwilson) [reply package.json]: Thanks. Sorry if there was confusion around the version; the change is done at release time by npm version command. You don't actually include that change in the PR directly.

File: HISTORY.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+2019-11-02 / 2.0.0
+==================
+
+  * deps: depd@2.0.0
+     - Drop support for Node.js 0.6 + 0.8
```
Comment (dougwilson) [inline HISTORY.md]: I think the 0.8 part is in the wrong area. Afaik depd 2 still supports 0.8, so if the desire is to drop 0.8 support, it should probably be listed at the level of this module's change (with the asterisk).

File: HISTORY.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
+2019-11-02 / 2.0.0
```
Comment (dougwilson) [inline HISTORY.md]: You can just put "unreleased" here or similar, otherwise this date will have to keep getting updated.

File: HISTORY.md (hunk 2 above)
Comment (arlac77) [reply HISTORY.md]: But the test failed for 0.8 too

Comment (dougwilson) [reply HISTORY.md]: That's strange, the `depd` has `>= 0.8` in the package.json (https://github.com/dougwilson/nodejs-depd/blob/master/package.json#L36) and also has a successful buld on Node.js 0.8 (https://travis-ci.org/dougwilson/nodejs-depd/jobs/602309636). What was the 0.8 failure? Maybe need to report up to the `depd` module the issue and look at resolving it.

The history entry for `depd` 2 only says 0.6 is dropped (https://github.com/dougwilson/nodejs-depd/blob/master/History.md#200--2018-10-26) not 0.8 like is written here.

Comment (arlac77) [reply HISTORY.md]: i reenabled 0.8 on travis and it works - sorry for the puzzle

**Comment by dougwilson** [comment]
Hi @arlac77 there is a `2.0` branch now and, since you deleted your original repository, I'm unable to update your PR to point to the 2.0 branch to land this PR. Unfortunately even if you re-create the repository, the PR has been disconnected from it. If you are interested in having your commits land, you're welcome to re-open it against the 2.0 branch.

**Pull request closed without merging**